tsl/src/nodes/decompress_chunk/decompress_chunk.c

```diff
@@ -1572,7 +1572,23 @@ create_compressed_scan_paths(PlannerInfo *root, RelOptInfo *compressed_rel, Comp
 
 	/* create parallel scan path */
 	if (compressed_rel->consider_parallel)
-		ts_create_plain_partial_paths(root, compressed_rel);
+	{
+		/* Almost the same functionality as ts_create_plain_partial_paths. However,
+		 * we create also a partial path for small chunks to encourage PostgreSQL
+		 * to choose a parallel plan.
```

Comment:
Otherwise, if there is no partial path for a single chunk, it can't make the parallel path for the entire Append, right?

Reply:
Probably makes sense to add this to the comment as well.

Reply:
This is correct. I updated the comment as suggested.